core_v2/sources/domains.move

```diff
@@ -628,14 +628,36 @@ module aptos_names_v2::domains {
     fun validate_registration_duration(
         registration_duration_secs: u64,
     ) {
-        assert!(registration_duration_secs % SECONDS_PER_YEAR == 0, error::invalid_argument(EDURATION_MUST_BE_WHOLE_YEARS));
+        assert!(
+            registration_duration_secs % SECONDS_PER_YEAR == 0,
+            error::invalid_argument(EDURATION_MUST_BE_WHOLE_YEARS)
+        );
 
         let num_years = (time_helper::seconds_to_years(registration_duration_secs) as u8);
         assert!(
             num_years > 0 && num_years <= config::max_number_of_years_registered(),
             error::out_of_range(EINVALID_NUMBER_YEARS)
         );
     }
+    public entry fun transfer_subdomain_owner(
```

Comment:
do we need to set reverse lookup in any case?

Reply:
oh you mean if it was someone's primary name we should remove it?